css/shopping-cart.css

```
@@ -676,4 +676,236 @@
     gap: 7px;
     margin: 0px 20px 0px 20px;
   }
+}
+
+
+/* Movil style */
+.can-interest__card{
+  position: relative;
+  border: 1px solid var(--color-solid);
+  background-color: var(--color-white);
+  padding: 0px 0px 25px 13px;
+  box-shadow: 0px 0px 4px 0px var(--box-shadowL);
+  border-radius: var(--eight-pixel); 
+}
+
+.can-interest__card:hover {
+  box-shadow: var(--box-shadowe);
+  border: 1.5px solid var(--color-solid);
+}
+
+.can-interest__card-link{
+  font-size: 0;
+  inset: 0;
+  position: absolute;
+  z-index: 1;
+}
+
+.can-interest__prod{
+  display: flex;
+  font-size: var(--sixteen-pixeles);
+  font-weight: var(--font-weight);
+  line-height: var(--line-height-small);
+  text-align: left;
+  color: var(--color-striking-gray);
+}
+
+.can-interest__price{
+  font-size: var(--twenty-pixel);
+  font-weight: var(--font-weight-bold);
+  line-height: var(--line-height-medium);
+  text-align: left;
+  color: var(--color-gray);
+  margin-bottom: 9px;
+}
+
+.can-interest__price--price-2{
```

Se ve algo raro lo de repetir la palabra en el modifier. Vi que lo tienes en varias partes. Quizas se vea menos extenso usando solo algo que describa que es lo que modifica, solo el numero, o el numero en letras
```suggestion
.can-interest__price--with-bottom-padding {
```